huaweicloud/services/gaussdb/data_source_huaweicloud_gaussdb_cassandra_instances.go

```diff
@@ -306,18 +307,22 @@ func dataSourceGeminiDBInstancesRead(d *schema.ResourceData, meta interface{}) e
 		backupStrategyList = append(backupStrategyList, backupStrategy)
 		instanceToSet["backup_strategy"] = backupStrategyList
 
-		//save geminidb tags
+		// save geminidb tags
 		if resourceTags, err := tags.Get(client, "instances", instanceID).Extract(); err == nil {
 			tagmap := utils.TagsToMap(resourceTags.Tags)
 			instanceToSet["tags"] = tagmap
 		} else {
-			logp.Printf("[WARN] Error fetching tags of geminidb (%s): %s", instanceID, err)
+			log.Printf("[WARN] error fetching tags of geminidb (%s): %s", instanceID, err)
 		}
 
 		instancesToSet = append(instancesToSet, instanceToSet)
 	}
 
-	d.SetId(hashcode.Strings(instancesIds))
+	uuid, err := uuid.GenerateUUID()
```

Comment:
if we generate the data source ID by GenerateUUID, then the `instancesIds` becomes unused ?
can we keep it as is?

Reply:
I understand, it has been restored.